examples/simple.py

```diff
@@ -1,9 +1,9 @@
 """A simple example that serializes a dictionary and prints out the result."""
 
-# isort: off
```

Comment:
Why'd you drop this? In user code, `asphalt.*` is always third-party, and removing this breaks that.